android/examples/Example_Application/README.md

```diff
@@ -8,16 +8,21 @@ This application provides a simple UI for encrypting/decrypting data with IronCo
 
 ### Prerequisites
 
-- Environment variable `ANDROID_SDK_ROOT` set to the Android Sdk install location
-- Android SDK 29. You can get the command line SDK [here](https://developer.android.com/studio) (scroll down to "Command line tools only") and use `sdkmanager` (found in tools/bin) to install additional prerequisites:
+- Install Android SDK 29. You can get the command line SDK [here](https://developer.android.com/studio) (scroll down to "Command line tools only").
+  - Note: **The extracted `tools` folder must follow a specific folder hierarchy. We recommend `AndroidCLI/cmdline-tools/tools`**.
+- Install Android 29 build and platform tools. This can be done with `sdkmanager` (found in `tools/bin`):
   ```bash
-    ./sdkmanager --sdk_root=PATH_TO_SDK_INSTALL_LOCATION --install "build-tools;29.0.3" "platform-tools" "platforms;android-29" "emulator"
+  ./sdkmanager "build-tools;29.0.3" "platform-tools" "platforms;android-29"
   ```
-- An Android Virtual Device (AVD)
+- Create the file `$HOME/.gradle/gradle.properties` and add the line `sdk.dir=PATH_TO_ANDROID_CLI_FOLDER`.
+- An Android emulator running, or a compatible Android phone connected.
+  - To start an emulator using the command line tools, follow these steps from the folder `AndroidCLI/cmdline-tools/tools/bin`:
+    1. `./sdkmanager "emulator" "system-images;android-29;google_apis;x86_64"`
+    1. `./avdmanager create avd -n pixel_3 -k "system-images;android-29;google_apis;x86_64" -d pixel_3`
+    1. `../../../emulator/emulator -avd pixel_3 -no-snapshot -noaudio -no-boot-anim`
 
 ### Directions
 
-1. Start the AVD.
 1. From the Example_Application root, run `./gradlew installDebug`.
```

Comment:
I didn't have ANDROID_SDK_ROOT defined which the build requires. Did I miss that step somewhere?
With `ANDROID_SDK_ROOT=/home/clintfred/Android ./gradlew installDebug` it did install properly.

Reply:
Good catch, I was thinking that setting the `sdk.dir` in your local `gradle.properties` (which is in the directions) would make it all work, but I was missing Chris's fix to make that happen. Did that now, so the current directions will work correctly.